docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,730 @@
+
+```bash
+$ cd /tmp/mii-sample/model-images
+$ ./imagetool/bin/imagetool.sh update \
+  --tag model-in-image:WLS-v1 \
+  --fromImage container-registry.oracle.com/middleware/weblogic:12.2.1.4 \
+  --wdtModel      ./model-in-image__WLS-v1/model.10.yaml \
+  --wdtVariables  ./model-in-image__WLS-v1/model.10.properties \
+  --wdtArchive    ./model-in-image__WLS-v1/archive.zip \
+  --wdtModelOnly \
+  --wdtDomainType WLS \
+  --chown oracle:root
+```
+
+If you don’t see the `imagetool` directory, then you missed a step in the prerequisites.
+
+This command runs the WebLogic Image Tool in its Model in Image mode, and does the following:
+
+  - Builds the final Docker image as a layer on the `container-registry.oracle.com/middleware/weblogic:12.2.1.4` base image.
+  - Copies the WDT ZIP file that’s referenced in the WIT cache into the image.
+      - Note that you cached WDT in WIT using the keyword `latest` when you set up the cache during the sample prerequisites steps.
+      - This lets WIT implicitly assume it’s the desired WDT version and removes the need to pass a `-wdtVersion` flag.
+  - Copies the specified WDT model, properties, and application archives to image location `/u01/wdt/models`.
+
+When the command succeeds, it should end with output like the following:
+
+```
+[INFO   ] Build successful. Build time=36s. Image tag=model-in-image:WLS-v1
+```
+
+Also, if you run the `docker images` command, then you will see a Docker image named `model-in-image:WLS-v1`.
+
+> Note: If you have Kubernetes cluster worker nodes that are remote to your local machine, then you need to put the image in a location that these nodes can access. See [Ensuring your Kubernetes cluster can access images](#ensuring-your-kubernetes-cluster-can-access-images).
+
+
+#### Create WebLogic domain
+
+In this section, you will deploy the new image to namespace `sample-domain1-ns`, including the following steps:
+
+- Create a namespace for the WebLogic domain.
+- Upgrade the operator to manage the WebLogic domain namespace.   
+- Create a Secret containing your WebLogic administrator user name and password.
+- Create a Secret containing your Model in Image runtime encryption password:
+    - All Model in Image domains must supply a runtime encryption Secret with a `password` value.
+    - It is used to encrypt configuration that is passed around internally by the operator.
+    - The value must be kept private but can be arbitrary; you can optionally supply a different secret value every time you restart the domain.
+- Deploy a Domain YAML file that references the new image.
+- Wait for the domain’s Pods to start and reach their ready state.
+
+##### Namespace
+
+Create a namespace that can host one or more domains:
+
+```bash
+$ kubectl create namespace sample-domain1-ns
+```
+
+##### Upgrade the operator
+
+Upgrade the operator to manage the WebLogic domains in namespace `sample-domain1-ns`.
```

Comment:
@rjeberhard wondering if this sample can show how to use labels so that the operator autodete cts the namespace and we do not need this step?

Reply:
Yes, you are absolutely correct. If back on line 110, the initial helm install used the new functionality to select namespaces by a label, then the helm upgrade would not be necessary here.

@sankarpn, do you understand or shall I propose a change to your branch?

Reply:
added these 2 params to helm install
  --set "domainNamespaceSelectionStrategy=LabelSelector" \
  --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \

Reply:
Can you remove this section of upgrading the operator to add the namespace of the domain?